pkg/openfeature/provider.go

```diff
@@ -5,6 +5,7 @@ import "errors"
 const (
 	DISABLED        string = "disabled"     // variant returned because feature is disabled
 	TARGETING_MATCH string = "target match" // variant returned because matched target rule
+	SPLIT						string = "split"			 	// variant returned because of pseudorandom assignment
```

Comment:
any idea what's going on with the spacing here?

Reply:
oops, fixed.